airflow/decorators/__init__.pyi

```diff
@@ -245,5 +245,121 @@ class TaskDecoratorCollection:
         :param cap_add: Include container capabilities
         """
         # [END decorator_signature]
+    def kubernetes(
+        self,
+        *,
+        python_callable: Optional[Callable] = None,
```

Comment:
Remove this since the argument-less form is not possible. See the `docker` decorator for an example.